### ISSUE TITLE
installation tests

### ISSUE THREAD

**Issue by pericsson** [PoonLab/sierra-local#55]
Hi, 
I tried running the installation tests at https://github.com/PoonLab/sierra-local/blob/master/INSTALL.md#for-macos-users but when i ran this code:
> **python3 scripts/retrieve_hivdb_data.py RT RT.fa**

I got this error message:
> urllib.error.HTTPError: HTTP Error 404: Not Found

I checked out the website that the python script was referring to and couldn't find the site so I tried to change it to the one I assumed was correct (or are these datasets found somewhere else?):
> **base_url = 'https://hivdb.stanford.edu/download/GenoRxDatasets'**

However, when I thereby ran:
> **sierralocal RT.fa**

I got another error message:
>searching path /Users/pericsson/miniconda3/lib/python3.7/site-packages/sierralocal/data/HIVDB*.xml
>Error: could not find local copy of HIVDB XML, attempting download...
>Unable to update HIVDB XML. Try manually downloading the HIVdb ASI2.
list index out of range
>/Users/pericsson/miniconda3/lib/python3.7/site-packages/sierralocal/data/apobec.tsv
>Error: could not retrieve APOBEC DRM data
>Unable to update APOBEC DRMs. Try manually downloading the APOBEC DRM TSV into data/apobec.tsv
>Traceback (most recent call last):
>  File "/Users/pericsson/miniconda3/bin/sierralocal", line 11, in <module>
>    exit_code = main.main()
>  File "/Users/pericsson/miniconda3/lib/python3.7/site-packages/sierralocal/main.py", line 181, in main
>    cleanup=args.cleanup, forceupdate=args.forceupdate)
>  File "/Users/pericsson/miniconda3/lib/python3.7/site-packages/sierralocal/main.py", line 108, in sierralocal
>    algorithm = HIVdb(path=xml, forceupdate=forceupdate)
>  File "/Users/pericsson/miniconda3/lib/python3.7/site-packages/sierralocal/hivdb.py", line 76, in __init__
>    self.root = xml.parse(str(self.xml_filename)).getroot()
>  File "/Users/pericsson/miniconda3/lib/python3.7/xml/etree/ElementTree.py", line 1197, in parse
>    tree.parse(source, parser)
>  File "/Users/pericsson/miniconda3/lib/python3.7/xml/etree/ElementTree.py", line 587, in parse
>   source = open(source, "rb")
>FileNotFoundError: [Errno 2] No such file or directory: 'None'

Do you know why I get this problem?
Thanks in advance!

**Comment by ArtPoon** [comment]
Hi @pericsson , thanks for the report!  Can you please send details on your OS version and version of Python?  Unfortunately we hadn’t done tests with Anaconda yet.

**Comment by pericsson** [comment]
Hi,
I'm using macOS Mojave version 10.14.6 and Python version 3.7.3.the HIVdb ASI2 and APOBEC DRM TSV from here https://hivdb.stanford.edu/assets/media/HIVDB_8.7.9e470b87.xml and here 
https://hivdb.stanford.edu/assets/media/apobec-drms.221b0330.tsv

**Comment by ArtPoon** [comment]
Reproduced error with macOS High Sierra and homebrew Python 3.7.4

**Comment by ArtPoon** [comment]
@pericsson thanks for the detailed info.  We tried to automate as much as possible to make it convenient for users to run the pipeline without having to manually download these resources, but of course we have no control when the Stanford database changes their web site access privileges (#58) or moved from static to dynamic web content (#59). 

I'm working on addressing these new issues (I'll probably have to add selenium to the dependency list) but you did the right thing in manually downloading the files.

Closing issue as this spawned new issues #56, #57, #58 and #59 - user has functioning program with manually obtained resource files.
